mypyc/ir/ops.py

```diff
@@ -1228,6 +1255,10 @@ def visit_unbox(self, op: Unbox) -> T:
     def visit_raise_standard_error(self, op: RaiseStandardError) -> T:
         raise NotImplementedError
 
+    @abstractmethod
+    def visit_c_function_call(self, op: CFunctionCall) -> T:
```

Comment:
If you rename the class, also rename everything else that has a name derived from the old name.